Fixes #73 : Event cards height and adds responsiveness to event carousel

shreyaskaundinya opened the pull request:
# Title
Fixes 
- Event card height and alignment
- Event card carousel responsiveness in tablet and mobile views
- Event Carousel arrow onClick

## Description
Alignment : 
![image](https://user-images.githubusercontent.com/42319389/137439398-58dc8740-f973-4e44-b3b7-bd65d40c6d2e.png)


Responsiveness :

Desktop 
Same as above

Tablet
![image](https://user-images.githubusercontent.com/42319389/137439806-47683da3-47db-4fd3-bd19-4be5d785ffbe.png)
![image](https://user-images.githubusercontent.com/42319389/137440005-e02726b0-2321-41ac-9a46-faaad64e544f.png)

Mobile
![image](https://user-images.githubusercontent.com/42319389/137439908-e39624cb-7d00-4f25-95d9-3c91fc79ac31.png)
![image](https://user-images.githubusercontent.com/42319389/137439922-11fd73ed-d677-425d-93a0-e4da67bed7bb.png)
![image](https://user-images.githubusercontent.com/42319389/137440060-bdda011e-29b8-4a63-8443-f2c43281e34a.png)



## Issue Closed
Closes #73 


## Notes [DELETE BEFORE SUBMITTING PR]
- Only submit **1** PR per issue and make only **1** Issue is solved per PR
- Make sure you've gone through the [contributing guidelines](https://github.com/acmpesuecc/acmpesuecc.github.io/blob/master/CONTRIBUTING.md).
- All your commits will be squashed into 1 commit during the merge
- The review process may take a while, be patient
- This section must be deleted before submitting the PR

shreyaskaundinya committed:
Fixes #73 : Event cards height and adds responsiveness to event carousel

arnxv0 commented:
![image](https://user-images.githubusercontent.com/57629464/137464893-584196f7-e490-4e86-8388-a28da3f38a34.png)

The logos on these cards are still not aligned.

shreyaskaundinya commented:
> 
> 
> ![image](https://user-images.githubusercontent.com/57629464/137464893-584196f7-e490-4e86-8388-a28da3f38a34.png)
> 
> The logos on these cards are still not aligned.

Will fix!

shreyaskaundinya commented:
Could i also fix the lazy loading? it doesn't seem to be working...

arnxv0 commented:
> Could i also fix the lazy loading? it doesn't seem to be working...

Sure, you might get some additional points for it too :)

shreyaskaundinya committed:
Fixes lazy loading of images

shreyaskaundinya committed:
Fixes :

- Member cards responsiveness
- Member cards github and email icon alignment
- Removal of  duplicate imports of bootstrap and font awesome

shreyaskaundinya commented:
Member cards 

Desktop
![image](https://user-images.githubusercontent.com/42319389/137541740-aebbd33a-1459-4415-9ff7-55cd985f9219.png)

Tablet
![image](https://user-images.githubusercontent.com/42319389/137541816-316c2609-c709-43a8-b243-3c73d0ba9cd8.png)

Mobile
![image](https://user-images.githubusercontent.com/42319389/137541855-a64b82c2-2e09-413b-804b-fb4fd1316586.png)

techverve commented:
![issue1](https://user-images.githubusercontent.com/53264615/137545176-cb805f27-1871-4071-9573-a10af64bf5a8.png)
the 5th core member picture is not inline

shreyaskaundinya commented:
That is because of overflow... The content wouldn't fit if I made it 5 in a row...
